doc/user_documentation.rst

```diff
@@ -481,6 +481,38 @@ Quick HOWTO for the `Package Review`_ time::
 .. _`Fedora Review`: https://pagure.io/FedoraReview
 .. _`Package Review`: https://fedoraproject.org/wiki/Package_Review_Process
 
+
+RPM Macros
+----------
+
+Copr defines custom RPM macros that are available for every build and
+can be used inside of a specfile.
+
+- ``%copr_username`` - Owner of the project, can be either a user or
+  group name, e.g. ``@copr``
+- ``%copr_projectname`` - Name of the project, e.g. ``copr-dev``
+- ``%vendor`` - This macro identifies the build system and namespace,
+  for example:
+
+    - ``Fedora Copr - user frostyx``
+    - ``Fedora Copr - group @copr``
+
+- ``%buildtag`` - This macro contains an ID of a Copr build,
+  e.g. ``.copr5925897``
+
+Macros for SRPM builds:
+
+- ``%dist`` - Regardless of the number of chroots that a build is
+  submitted for, the SRPM package is built only once. And its
+  ``%dist`` tag would be determined by the operating system on a
+  builder. To avoid such confusion, we redefine ``%dist`` to ``%nil``
+  for SRPM builds.
```

Comment:
can you point at the issue @nikromen is dealing with?  this is a bit off-topic though as %dist is not defined by Copr -- or maybe, we should point out that "we try to undefine it as much as possible" to be distro-agnostic.